src/components/message-box/_message_box.scss

```diff
@@ -0,0 +1,24 @@
+$includeHtml: false !default;
+
+@if ($includeHtml) {
+  .sg-message-box {
+    position: relative;
+    display: inline-block;
+
+    &__close {
+      position: absolute;
+      right: 0;
+      top: 0;
+      padding: gutter(1 / 2);
+      cursor: pointer;
+      z-index: 1;
+   }
+  }
+
+  .sg-box {
```

Comment:
This should be part of box styles

Reply:
i thought of that but wasn't sure - changing now

Reply:
:+

Reply:
@vergilius but this is specific style to be used just by message box